### DISCUSSION TITLE
Default Server?

### DISCUSSION THREAD

**Discussion by PaxVobiscuit** [Taxel/PlexTraktSync#635] [Q&A] [answered]
How do I change the "default" server to something other than the one I chose when I first ran the tool?

**Comment by simonc56** [comment, marked as answer]
Try `login` command or remove `.env` file from your config dir.

**Reply by glensc** [reply]
`plextraktsync login` command only starts wizard if the configuration is missing. so you have to remove `.env` and then `plextraktsync login`, or use the other command:
- https://github.com/Taxel/PlexTraktSync/discussions/635#discussioncomment-1781998

**Comment by glensc** [comment]
Run `plextraktsync trakt-login` command, it will run the wizard allowing you pick server again.

**Comment by sabonim38** [comment]
I have to reinstall my plexserver - and add a new name. Now everything I try to run plextrakt I have to use the choose the default server. I have tried to run the command: plextraktsync trakt-login, and everything goes fine.
But when I try to run python3 -m plextraktsync, I have to choose the default server again.

I have tried to delete and install plextrakt and also create a new key. Nothing works....

Any help?

**Reply by glensc** [reply]
lack of details of your system. most likely `plextraktsync` and `python3 -m plextraktsync` execute different installation, and different config. stick to once form of execution, use `info` subcommand to locate your config path.
